integration/vscode/ada/syntaxes/ada.tmLanguage.json

```diff
@@ -183,13 +183,26 @@
 				"7": { "patterns": [ { "include": "#exponent_part" } ] }
 			}
 		},
+		"declarative_item": {
+			"patterns": [
+				{ "include": "#body" },
+				{ "include": "#basic_declarative_item" }
+			]
+		},
```

Comment:
This part doesn't work. Whatever the way we put it (`body` first or `basic_declarative_item` first), either `function_body` or `function_specification` is matched.
Maybe the solution is to merge the two definitions. @Entomy you may have better solution.

Reply:
I just got back to where I have my own computer. I'll look at this in about an hour.